src/service_name.json

```diff
@@ -874,11 +864,6 @@
     "AzureServiceName": "Event Grid",
     "URL": "https://learn.microsoft.com/en-us/azure/event-grid"
   },
-  {
-    "Command": "az mariner-baremetal",
-    "AzureServiceName": "Mariner BareMetal Platform",
-    "URL": ""
-  },
   {
```

Comment:
`vm-repair` and `hybridaks` not covered, refer to this: https://github.com/Azure/azure-cli-extensions/pull/8455/files